app/src/main/java/org/fossasia/openevent/events/TracksDownloadEvent.java

```diff
@@ -0,0 +1,20 @@
+package org.fossasia.openevent.events;
+
+/**
+ * Created by MananWason on 8/4/2015.
+ */
+public class TracksDownloadEvent {
```

Comment:
It can be refactored but then we won't be able to distinguish between events. Let me see if we can do something about this.